artifacts/deploy/karmada-apiserver.yaml

```diff
@@ -54,10 +54,10 @@ spec:
             - --service-account-key-file=/etc/kubernetes/pki/karmada.key
             - --service-account-signing-key-file=/etc/kubernetes/pki/karmada.key
             - --service-cluster-ip-range=10.96.0.0/12
-            - --proxy-client-cert-file=/etc/kubernetes/pki/karmada.crt
-            - --proxy-client-key-file=/etc/kubernetes/pki/karmada.key
+            - --proxy-client-cert-file=/etc/kubernetes/pki/front-proxy-client.crt
+            - --proxy-client-key-file=/etc/kubernetes/pki/front-proxy-client.key
```

Comment:
If we change the parameter:
@prodanlabs we should update `karamda init` right?
@jrkeen we should update `chart` right?

Reply:
You might be confused about this PR. We'll talk about this PR at tomorrow's community meeting.

Reply:
Yes, the chart should be updated.

Reply:
Yes, i follow up